### ISSUE TITLE
Debug images only showing one column and gutter

### ISSUE THREAD

**Issue by rctneil** [ericam/susy#480]
Here's my $susy grid settings:

```
$susy: (
  flow: ltr,
  math: fluid,
  output: float,
  gutter-position: after,
  container: 1440px,
  container-position: center,
  columns: 12,
  gutters: .25,
  column-width: false,
  global-box-sizing: border-box,
  last-flow: to,
  debug: (
    image: hide,
    color: rgba(#66f, .25),
    output: background,
    toggle: top right,
  ),
  use-custom: (
    background-image: true,
    background-options: false,
    box-sizing: true,
    clearfix: false,
    rem: true,
  )
);
```

For some reason I only get 1 pair of column and gutter background debug images showing up.  Look at the screenshot.

Any ideas?

![screen shot 2015-05-08 at 13 00 11](https://cloud.githubusercontent.com/assets/774459/7535924/65c2c9e6-f582-11e4-921c-11945934ac83.png)

**Comment by ericam** [comment]
The debug image uses that single-column gradient, along with `background-size` and `background-repeat` to create the grid. It looks like you might be overriding those two properties somewhere else in your css.

**Comment by rctneil** [comment]
@ericam Ok,

That makes sense. I looked at the in use styles in the Chrome Inspector and it's showing a background-size line of this:

`background-size: 8.47458%, 100% 1.2;`

It seems it's colliding with the `$base-line-height` variable. If I rename `$base-line-height` to something totally different then the background size outputs correctly and the column debug image looks perfect.  I have actually had this before but this is an extremely cut down application and it still happens.

Any thoughts?

**Comment by ericam** [comment]
How are you calling the background? If you do it in settings, use:

```scss
$susy: (
  debug: (
    image: columns,
  ),
);
```

If you do it in a mixin, use the `columns` keyword instead of the `show` keyword.

**Comment by rctneil** [comment]
Hey,

I've tried both options by using the `columns` keyword and it still doesn't work.  The `$base-line-height` variable is still interfering and giving that odd output.

If I rename that variable it all outputs fine, it's only if I have a variable with that exact name.  Just for note. My variable is defined like so: `$base-line-height: 1.2;`

Thanks. I hope we can resolve this.

**Comment by rctneil** [comment]
Anymore thoughts on this?

**Comment by ericam** [comment]
Looking into it...

**Comment by ericam** [comment]
Sorry, I had that wrong. The keyword should be `show-columns` not `columns`. We should also find a way to be smarter about checking for Compass integration, so I'll leave this issue open.

**Comment by rctneil** [comment]
I have had this occur again in another project and it was a brand new project with only Susy and Bourbon installed.  It seems to do this when the `$base-line-height` variable is set. As soon as you rename that variable to something else it's fine and the grid outputs correctly.  No Compass used in either project.

**Comment by ericam** [comment]
Right. That's exactly the same issue, and the same fix will work. Use `show-columns` instead of `show` to see your grid.

Susy is looking for `$base-line-height` because that's what Compass uses. But you aren't using Compass, so it's breaking. We need to be smarter about checking for Compass.

**Comment by houd** [comment]
Hi.
I have the same issue as @rctneil  trying to use Susy with Foundation Framework. It was fixed by commenting `$base-line-height: 1.5 !default;` in _global.scss foundation component.
It may help someone else.

**Comment by ericam** [comment]
Fixed in 433a0bad2da8bbd0d5ecde8ad2dc6e7260c4aa2d
